apps/roam/src/utils/createReifiedBlock.ts

```diff
@@ -0,0 +1,118 @@
+import createBlock from "roamjs-components/writes/createBlock";
+import createPage from "roamjs-components/writes/createPage";
+import getPageUidByPageTitle from "roamjs-components/queries/getPageUidByPageTitle";
+import setBlockProps from "./setBlockProps";
+import { getSetting } from "~/utils/extensionSettings";
+
+const queryForReifiedBlocksUtil = async (
+  parameterUids: Record<string, string>,
+): Promise<[string, Record<string, string>][]> => {
+  const paramsAsSeq = Object.entries(parameterUids);
+  const query = `[:find ?u ?d
+  :in $ ${paramsAsSeq.map(([k]) => "?" + k).join(" ")}
+  :where [?s :block/uid ?u] [?s :block/props ?p] [(get ?p :discourse-graph) ?d]
+  ${paramsAsSeq.map(([k]) => `[(get ?d :${k}) ?${k}]`).join(" ")} ]`;
+  return (await window.roamAlphaAPI.data.async.q(
+    query,
+    ...paramsAsSeq.map(([, v]) => v),
+  )) as [string, Record<string, string>][];
+};
+
+export const queryForReifiedBlocks = async (
```

Comment:
Why is this exported?  What is the use case based on the existing task?

Reply:
Actually, this isn't used anywhere currently, is it.  If it's not being used, let's remove it.